### PR TITLE
**Breaking:** CON-1281 Introduce more constants for primary color variations

### PR DESCRIPTION
**Pull request by mpotomin**
# Summary
Add more constants that represent primary color variations

# Related issue
CON-1281

# To be tested

Me
- [x] No error or warning in the console on `localhost:6060`

Tester 1

- [ ] Things look good on the demo.
  

Tester 2

- [ ] Things look good on the demo.

### CODE REVIEW

File: src/utils/constants.ts [hunk 1]
```diff
@@ -84,6 +84,7 @@ const textColors = {
  * A specialized color palette for borders.
  */
 const borderColors = {
+  primary: setAlpha(0.1)(primaryColor),
```
Comment (stereobooster) [inline src/utils/constants.ts]: We already have:

```ts
const color = {
  primary: primaryColor,
```

the same name would be confusing

**Comment by stereobooster** [comment]
Do you want to consolidate all occurrences of `setAlpha(0.05)(theme.color.primary)`?

File: src/utils/constants.ts (hunk 1 above)
Comment (mpotomin) [reply src/utils/constants.ts]: Sure, what would you suggest instead?

Comment (stereobooster) [reply src/utils/constants.ts]: Hm 🤔 `lightPrimary` maybe? (there is darkPrimary)
Or maybe "accent" color?

Comment (mpotomin) [reply src/utils/constants.ts]: We already have the `highlight`, that is 5% alpha of the `primary`, so let's try `accent` for 10% alpha of the `primary`, that we could occasionally use for borders as well

Comment (stereobooster) [reply src/utils/constants.ts]: `highlightBorder` maybe?

Comment (mpotomin) [reply src/utils/constants.ts]: Sounds too specific for the border. There are cases in the product where we need it not for the border only. Shall we settle for `constants.color.accent `?

Comment (mpotomin) [reply src/utils/constants.ts]: Just brought it up in the team sync, agreed to use `constants.color.accent` in this case. See [this ticket](https://contiamo.atlassian.net/browse/CON-1281)

**Comment by mpotomin** [comment]
@stereobooster I think I've addressed all the open discussions. Please take a look again 🙏

File: src/utils/constants.ts [hunk 2]
```diff
@@ -122,6 +122,7 @@ const color = {
   ghost: "hsla(0, 0%, 100%, 0.33)",
   white: whiteColor,
   highlight: setAlpha(0.05)(primaryColor),
+  accent: setAlpha(0.1)(primaryColor),
```
Comment (stereobooster) [inline src/utils/constants.ts]: I wonder if this will work with dynamically changed `primaryColor` via theme 🤔

Comment (mpotomin) [reply src/utils/constants.ts]: Ah snap, I guess it won't 😭

Comment (mpotomin) [reply src/utils/constants.ts]: Should be fixed now

File: src/utils/constants.ts [hunk 3]
```diff
@@ -121,8 +121,6 @@ const color = {
   basic: "#636363",
   ghost: "hsla(0, 0%, 100%, 0.33)",
   white: whiteColor,
-  highlight: setAlpha(0.05)(primaryColor),
```
Comment (stereobooster) [inline src/utils/constants.ts]: this can be a breaking change

Comment (mpotomin) [reply src/utils/constants.ts]: Good point, updated the description

**Pull request merged**